godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Vector2.kt

```diff
@@ -0,0 +1,311 @@
+@file:Suppress("unused", "MemberVisibilityCanBePrivate")
+
+package godot.core
+
+import godot.gdnative.*
+import kotlinx.cinterop.*
+import kotlin.math.*
+
+class Vector2(var x: RealT, var y: RealT) : Comparable<Vector2>, CoreType {
+    //CONSTANTS
+    companion object {
+        const val AXIS_X: Int = 0
```

Comment:
Can we use enums for `AXIS_*` values?

Reply:
It was initially one but I tried to stick GDScript as much as possible. And there is no enum in GdScript as you already know.

Reply:
I don't think we need to limit ourselves with GDScript on this one.